components/webac/src/main/java/org/trellisldp/webac/WebACService.java

```diff
@@ -96,10 +96,13 @@
     private final boolean checkMembershipResources;
 
     /**
-     * Create a WebAC-based authorization service.
+     * No-op for CDI. Create a WebAC-based authorization service.
      */
     public WebACService() {
-        this(getDefaultResourceService());
+        //this(getDefaultResourceService());
+    	this.resourceService = null;
```

Comment:
Is this actually a safe way to build a `WebACService`? Will it work if this ctor is used?

Reply:
It would be better to leave this line as it is: `this(getDefaultResourceService());`

Reply:
I would remove all of the changes to this constructor.

Reply:
If I do that, then I will have to remove the null checks, since getDefaultResourceService() returns a null in the CDI proxy context.